libraries/botbuilder-ai/tests/qnaMaker.test.js

```diff
@@ -778,6 +780,105 @@ describe('QnAMaker', function () {
             assert(typeof fetch === 'function');
         });
     });
+
+    describe('Active Learning', function() {
+        it('executeHttpRequest() should return JSON result from Train API response of 204 No-Content', function() {
+            const { endpoint, trainApi, feedbackRecords, successfullyTrainedResult } = new TrainApiHelper();
+
+            async () => await executeHttpRequestToTrainApi();
+        
+            async function executeHttpRequestToTrainApi() {
+                nock(endpoint.qnaHost)
+                    .post(trainApi)
+                    .reply(204);
+                
+                const payloadBody = JSON.stringify({
+                    feedbackRecords: feedbackRecords
+                });
+
+                try {
+                    const httpUtils = new HttpRequestUtils();
+                    const qnaResult = await httpUtils.executeHttpRequest(
+                        endpoint.qnaHost + trainApi,
+                        payloadBody,
+                        endpoint
+                    );
+                    
+                    assert.deepStrictEqual(qnaResult, successfullyTrainedResult);
```

Comment:
If the `executeHttpRequestToTrainApi()` returns `httpUtils.executeHttpRequest()`, you can evaluate the `qnaResult` outside of the helper function and simplify this code.

Reply:
removed helper function executeHttpRequestToTrainApi()